src/components/FeaturePanel/Runways/loadRunways.ts

```diff
@@ -0,0 +1,40 @@
+import { fetchText } from '../../../services/fetch';
+
+export type Runway = {
+  id: string;
+  type: string;
+  ref?: string;
+  width?: string;
+  length?: string;
+  surface?: string;
+};
+
+export async function loadRunways(id: string) {
+  const query = `[out:csv(::id, ::type, ref, width, length, surface; false; '||')];
+  wr(${id});
+  map_to_area;
+  way
+    [aeroway=runway]
+    (area);
```

Comment:
This query doesn't work for node airports. Can we find a way to do it? eg. https://osmapp.org/node/700722944